variable_distance_exploration/lib/experimental_media_query.dart

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/widgets.dart';
+
+/// Interaction models.
+enum DeviceCorporealContext {
+  /// Viewed only.
+  ambient,
+
+  /// Interactive.
```

Comment:
why isn't this field called interactive?

Reply:
We're working on refining the vocabulary. This is what the designers were using at the moment tho. So I aligned with them until we agree on more permanent vocab.